src/sentry/integrations/vsts/integration.py

```diff
@@ -110,30 +91,28 @@ def get_pipeline_views(self):
 
         return [
             identity_pipeline_view,
-            ProjectConfigView(),
         ]
 
     def build_integration(self, state):
         data = state['identity']['data']
         account = state['identity']['account']
         instance = state['identity']['instance']
-        project = state['project']
 
         scopes = sorted(VSTSIdentityProvider.oauth_scopes)
         return {
-            'name': project['name'],
-            'external_id': project['id'],
+            'name': account['AccountName'],
+            'external_id': account['AccountId'],
```

Comment:
So with this change we'll be associating an integration directly to a users account right? Or am I mistaking account again for like "the organization account" type thing.

Reply:
Hi @EvanPurkhiser , so currently I was thinking that was the best policy for it. But I'm open to suggestions. I was just trying to get something that made a bit more sense. The account in this case refers to the VSTS workspace. So for instance there's this example in the [docs](https://docs.microsoft.com/en-us/rest/api/vsts/account/accounts/list?view=vsts-rest-4.1#account):
```json
{
  "count": 2,
  "value": [
    {
      "accountId": "6affcbef-fe5d-44c6-8a29-439b79aaa118",
      "accountUri": "https://Fabrikam-Fiber-Inc.vssps.visualstudio.com:443/",
      "accountName": "Fabrikam-Fiber-Inc",
      "properties": {}
    },
    {
      "accountId": "bf83696f-383b-4980-95b5-1f1127640a8a",
      "accountUri": "https://NormalPaulk.vssps.visualstudio.com:443/",
      "accountName": "NormalPaulk",
      "properties": {}
    }
  ]
}
```
So it would be for example, `"NormalPaulk"`, would be the `accountName`. Accounts can have many projects as seen below:
<img width="1680" alt="screen shot 2018-06-12 at 6 02 09 pm" src="https://user-images.githubusercontent.com/12959365/41324715-04f30a76-6e6b-11e8-9fbd-13244d85a7ea.png">

I think though the actual `user_identity` should probably be a Microsoft account. Something to that affect. I can look into what the options are there. 

But yeah, I understand now what the concepts of `Identity`/`Integration` are used for, and I didn't at the time when this was written. I thought this would work better, but I'm open to suggestions. Let me know what you think!

Reply:
Yeah the user_identity should definitely be a Microsoft account if it can be. We can probably sync up with @adhiraj for that too.

Reply:
Yea we'll need to update this in a couple of weeks or so when we start implementing msft login